README.md

```diff
@@ -1,38 +1,36 @@
 # mame2003-libretro
-MAME as it was in 2003, and using the libretro API. Accepts romset **MAME 0.78**. Any file-not-found issues will be because your romset is wrong, or incomplete (e.g. you're trying to run a clone .zip, without the parent .zip present).
+MAME as it was in 2003, using the libretro API. Suitable for lower-end devices that would struggle to run current versions of MAME (later versions of MAME are increasingly accurate, thus can perform worse).
 
-Suitable for lower-end devices that would struggle to run current versions of MAME (later versions of MAME are increasingly accurate, thus can perform worse).
+**Note**: The MAME 2003 core accepts **MAME 0.78** ROMs. Each version of an arcade emulator must be used with ROMs that have the same exact version number. For example, MAME 0.37b5 ROMsets are required by the MAME 2000 emulation, but MAME 0.37b5 sets will not work correctly with the MAME 2003 or MAME 20010 emulator cores, which requires MAME 0.78 and MAME 0.139 ROM sets.
 
-Has support for a single mouse (or touch device) in games that support trackpads, etc (via the "Mouse Device" core option)
-Supports one or two spinners/dials (via the "Share 2 player dial controls across one X/Y device" core option)
+File-not-found errors are the result of a ROMset that is wrong or incomplete (e.g. you're trying to run a clone .zip, without the parent .zip present).
 
-Tested for OS X, linux (including Raspberry Pi 2, 3), Wii, and Android (arm-v7a).
+MAME 2003 has support for multiple mice (or touch device) in games that support trackpads, etc. MAME 2003 also supports one or two spinners/dials (via the "Share 2 player dial controls across one X/Y device" core option).
 
 ### Directories
-Generates directories as it requires:
-* User-generated content are in sub-directories within `/libretro savefile dir/mame2003/` e.g.:
+* Some MAME games require data from an internal hard drive, CD-ROM, laserdisk, or other media in order to be emulated -- those forms of media are packaged as CHD files. CHD files should be copied to subfolders within the folder where the MAME ROM zips have been installed. e.g.:
 ```
-/libretro savefile dir/mame2003/diff/
-/libretro savefile dir/mame2003/nvram/
-/libretro savefile dir/mame2003/hi/
-/libretro savefile dir/mame2003/cfg/
-/libretro savefile dir/mame2003/inp/
-/libretro savefile dir/mame2003/memcard/
-/libretro savefile dir/mame2003/snap/
+/libretro content dir/blitz/blitz.chd
+```
+* Some games require an additional zip file with recorded sounds or music in order for audio to work correctly. Audio 'sample' files should be placed in subdirectories within `/libretro system dir/mame2003/` e.g.:
+```
+/libretro system dir/mame2003/samples/
 ```
-* .dat files should be placed within `/libretro system dir/mame2003/` e.g.:
+* High schore, cheat, and history metadata files should be placed within `/libretro system dir/mame2003/` e.g.:
 ```
```

Comment:
schore > score

maybe better to have something like
"should be moved from the /metadata/ folders to ..."